test/acceptance/renumber.js

```diff
@@ -0,0 +1,12 @@
+describe('todo renumber', function() {
```

Comment:
Not to speak for @vesln, but it looks like `acceptance` is testing the `cli` where `test/*.js` is testing the domain objects.